packages/phoenix-ng/projects/phoenix-app/cypress/e2e/cms.test.ts

```diff
@@ -0,0 +1,171 @@
+describe('CMS', () => {
+  it('should match the whole page', () => {
+    cy.visit('/cms').then(() => {
+      cy.contains('This may take a while').should('not.be.visible');
+      // in order to make the detector appear which loads in 2 - 2.5 seconds
+      cy.wait(2500);
+      cy.document().toMatchImageSnapshot();
+    });
+  });
+
+  describe('Phoenix Menu', () => {
+    it('should have options to Save and Load state', () => {
+      cy.get('.phoenix-menu-item-right .item-settings').click();
+
+      cy.contains('Save state').should('exist');
+      cy.contains('Load state').should('exist');
+
+      cy.get('.phoenix-menu-item-right .item-config-backdrop').click();
+    });
+
+    it('should display correct options under event data dropdown', () => {
+      cy.get('.phoenix-menu-item-right > button').eq(0).click();
+      cy.get('.phoenix-menu-item-right > button').eq(1).click();
+
+      cy.contains('Tracks').should('exist');
```

Comment:
You can scope this first: `cy.get("app-phoenix-menu").contains('Tracks').should('exist');`

This is true for the 4 selectors here. Because `Tracks` is a word that could appear somewhere else later if we add something.